zlp-scheduler/app/controllers/admin_controller.rb

```diff
@@ -220,4 +220,34 @@ def student_actions
     @actions = StudentAction.all.order(created_at: :desc)
   end
   
+  def add_student
+    case request.method_symbol    
+      when :get
+        @user=User.new()
+        session[:cohort_id]=params[:cohort_id]
+        @cohort_name=Cohort.find_by(id:params[:cohort_id]).name
+        @cohort_id=params[:cohort_id]
+    
+      when :post
+        if User.find_by(email:params[:user][:email]).nil?
+          @user = User.new()
+          @user.role = 'student'
+          @user.firstname=params[:user][:firstname]
+          
+          @user.lastname=params[:user][:lastname]
+          @user.email=params[:user][:email]
+          @user.cohort_id = session[:cohort_id]
+          @user.uin = params[:user][:uin]
+          @user.password='Temp'
```

Comment:
You can set `@user.activate = false`, so new students cannot use 'Temp' to login before claim account.

Reply:
Fixed